### PR TITLE
update instagram followers count

### PR DESCRIPTION
**Pull request by divyanshu1810**
# Description

Instagram followers count update

### CODE REVIEW

File: pages/api/instagram.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import axios from 'axios'
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+export default async function handler(
```
Comment (shawshankkumar) [inline pages/api/instagram.ts:5]: Why not do this sever side?

**Comment by divyanshu1810** [comment]
### If data API worked and data fetched properly
image
### else
image

File: pages/services.tsx [hunk 2]
```diff
@@ -125,3 +129,28 @@ const Services = () => {
 }
 
 export default Services
+
+export async function getServerSideProps() {
+  try {
+    const response = await axios.get(
+      'https://www.srmkzilla.net/api/instagram',
```
Comment (shawshankkumar) [inline pages/services.tsx:136]: Why would you call the api itself? Why not just write the logic you wrote in /api here? both run on server

File: pages/services.tsx [hunk 3]
```diff
@@ -82,7 +82,11 @@ const Services = () => {
                 </h3>
               </Fade>
               {/* <Numbers /> */}
-              <Stats />
+              <Stats
+                followers={
+                  props.data?.data.data.user.edge_followed_by.count || 3720
```
Comment (shawshankkumar) [inline pages/services.tsx:87]: add optional chaining `?` to all so that this doesn't break.

File: pages/api/instagram.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```
Comment (shawshankkumar) [inline pages/api/instagram.ts:1]: node needed

**Review by shawshankkumar — CHANGES_REQUESTED** [review]
@divyanshu1810

File: pages/services.tsx (hunk 2 above)
Comment (divyanshu1810) [reply pages/services.tsx:136]: done

File: pages/services.tsx (hunk 3 above)
Comment (divyanshu1810) [reply pages/services.tsx:87]: done

File: pages/api/instagram.ts (hunk 4 above)
Comment (divyanshu1810) [reply pages/api/instagram.ts:1]: done

**Review by shawshankkumar — APPROVED** [review]
lgtm!

**Pull request merged**